changelog

```diff
@@ -2,6 +2,7 @@ MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-andr
 
 Version v.Next
 ----------
+- Expose RT to Broker apps.
```

Comment:
You may want to say broker refresh token here explicitly.  As well as indicate why this support is needed.  Once it's shipped any broker app will be able to use this... in any scenario, but it would be good to know which scenario originally required it.